src/fileviews/ufileviewheader.pas

```diff
@@ -232,7 +232,7 @@ procedure TFileViewHeader.onKeyRETURN(Sender: TObject);
   NewPath: String;
   AClass: TFileSourceClass;
 begin
-  NewPath:= NormalizePathDelimiters(FPathEdit.Text);
+  NewPath:= mbExpandFileNameForLocalFile(FPathEdit.Text);
   NewPath:= ReplaceEnvVars(ReplaceTilde(NewPath));
```

Comment:
Remove `ReplaceEnvVars` from here, it is already called in the `mbExpandFileName`.